cluster/examples/kubernetes/ozone/install-operator-rbac.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: objectstores.ozone.rook.io
+spec:
+  group: ozone.rook.io
+  names:
+    kind: ObjectStore
+    listKind: ObjectStoreList
+    plural: objectstores
+    singular: objectstore
+  scope: Namespaced
+  version: v1alpha1
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rook-ozone-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-ozone-operator
+  namespace: rook-ozone-system
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: rook-ozone-operator
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  - secrets
+  - pods
+  - services
+  - configmaps
+  verbs:
+  - get
+  - watch
+  - create
+  - update
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  - daemonsets
+  verbs:
+  - get
+  - create
+  - update
+- apiGroups:
+  - ozone.rook.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+- apiGroups:
+  - rook.io
```

Comment:
You likely don't need the `rook.io` apiGroup. I believe this is only for the rook flex driver, which you wouldn't be using.

Reply:
Thanks, removed.